EIPS/eip-1682.md

```diff
@@ -0,0 +1,208 @@
+---
+eip: 1682
+title: Storage Rent
+author: Felix J Lange (@fjl), Martin Holst Swende (@holiman)
+discussions-to: <URL>
```

Comment:
Needs a discussion URL.

Reply:
Apparently this is the correct one: https://ethereum-magicians.org/t/storage-rent-eip/2357